devices.js

```diff
@@ -79,6 +81,55 @@ const waveshare7in2v2Vertical = {
         this.driver.init();
     },
 };
+const renderCanvas = () => {
```

Comment:
@ascension can you provide a little more information about what this method is for? It seems like it's just writing the text 'Awesome!'?